doc/make-release.md

```diff
@@ -1,22 +1,20 @@
-# How to release Opencast Studio
+# How to cut a release for Opencast
 
-Checkout the branch `master` from Opencast Studio and make sure it is up-to-date (working directory should be clean!).
-Execute `./create-release.sh`. Now there should be two new `*.tar.gz` files.
-
-## Release
-
-Go to [the release section on GitHub](https://github.com/elan-ev/opencast-studio/releases) and click "Draft a new release". Click on "choose a tag" and enter the current date in YYYY-MM-DD and then "Create new tag".
-Make sure the target is still `master`.
-As a release title, enter the current date again in the same format.
-List all changes (since the last release) in the text box.
-Upload the two `*.tar.gz` files and click "Publish release".
-
-## Update Opencast
-
-Checkout a new branch.
-Go to `modules/studio/pom.xml`. You have to change two lines:
-- `<opencast.studio.url>`: change the url to the new relase-url from `*-integrated.tar.gz`. (Usually only the date has to be adjusted here.)
-- `<opencast.studio.sha256>`: update the sha256 hash. (Run `sha256sum *-integrated.tar.gz` in Opencast Studio folder)
- Open a new pull request (optional: add a link to the release and changelog in the description)
-
- Before opening a new pull request, you can build Opencast and test Opencast Studio. The release date should be on the info page.
+1. Switch to the commit you want to turn into the release
+2. Create and push a new tag
+   ```bash
+    DATE=$(date +%Y-%m-%d)
+    git tag -m Release -s "$DATE"
+    git push upstream "$DATE":"$DATE"
+   ```
+3. Wait for the [Create release draft](https://github.com/opencast/opencast-studio/actions/workflows/create-release.yml)
+   to finish
+    - it will create a new [GitHub release draft](https://github.com/opencast/opencast-studio/releases)
+    - review and publish the draft
+4. Submit a pull request against Opencast
+    - [Update the release](https://github.com/opencast/opencast/blob/b2bea8822b95b8692bb5bbbdf75c9931c2b7298a/modules/studio/pom.xml#L16-L17)
```

Comment:
This could more explicitly mention to update the URL to the tar.gz and the sha256 hash. And also say how to get the hash (as mentioned in the previous version of this document)

Reply:
You are right. 
If I had to do a release for the first time, this maybe would not be clear enough. 

A few infos are better than none, right? :D